src/interface/src/app/types/region.types.ts

```diff
@@ -27,3 +27,22 @@ export function regionToString(region: Region | null): string {
   }
   return 'sierra-nevada';
 }
+
+export const RegionsWithString = [
```

Comment:
not a fan of this, I think we need to rework how we deal with regions in general, but I dont want to bite that bullet on this pr

Reply:
Fair point -- I'm also not sure how we should deal with these, especially if/when these expand.

Reply:
Well Im thinking something like this:
```
export type Region =
  | 'sierra-nevada'
  | 'southern-california'
  | 'central-coast'
  | 'northern-california';

export const Regions: Record<Region, string> = {
  'sierra-nevada': 'Sierra Nevada',
  'central-coast': 'Central Coast',
  'northern-california': 'Northern California',
  'southern-california': 'Southern California',
};

```

where the key is the actual value that the backend uses, and we have a record with the literal display string.  No regionToString function or extra stuff. But...that would require a major refactor